Documentation/ceph-filesystem-crd.md

```diff
@@ -51,6 +53,41 @@ spec:
     #    memory: "1024Mi"
 ```
 
+(These definitions can also be found in the [`filesystem.yaml`](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/filesystem.yaml) file)
```

Comment:
Pardon my ignorance here, but will this renderered properly when the doc gets made? My attempts to show it right via this patch didn't show the hyperlink properly.

Reply:
Yes, this will be rendered properly when viewed on https://rook.io.
You would need to use the `build/run.sh` script in the rook.github.io repo. It runs Jekyll by container locally and you can view the page then, see README.md for instructions.